requirements-dev.txt

```
@@ -1,7 +1,8 @@
 ninja
 flaky
 numpy
-git+https://github.com/PennyLaneAI/pennylane.git@master
+# git+https://github.com/PennyLaneAI/pennylane.git@master
+pennylane~=0.28
```

If merging to master, this needs to be changed:
```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```